packages/web-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorNeutralForeground3,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+} from '../theme/design-tokens.js';
+
+/** RadioGroup styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 2px 0;
+  }
+  ::slotted([slot='label']) {
+    color: ${colorNeutralForeground3};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
```

Comment:
Looks like your left and right paddings are 2 different values, use `padding-inline` instead